azure/Guidelines.md

```diff
@@ -907,45 +903,59 @@ For a PUT (create or replace) with additional long-running processing:
 
 <a href="#lro-status-monitor-retry-after" name="lro-status-monitor-retry-after">:white_check_mark:</a> **DO** include a `Retry-After` header in the response to GET requests to the status monitor if the operation is not complete. The value of this header should be an integer number of seconds to wait before making the next request to the status monitor.
 
-<a href="#lro-status-monitor-post-action-result" name="lro-status-monitor-post-action-result">:white_check_mark:</a> **DO** include the `result` property (if any) in the status monitor for a POST action-type long-running operation when the operation completes successfully.
+<a href="#lro-status-monitor-post-action-result" name="lro-status-monitor-post-action-result">:white_check_mark:</a> **DO** include the `result` property (if any) in the status monitor for a POST-Action long-running operation when the operation completes successfully.
 
-<a href="#lro-status-monitor-no-resource-result" name="lro-status-monitor-no-resource-result">:no_entry:</a> **DO NOT** include a `result` property in the status monitor for a long-running operation that is not a POST action-type long-running operation.
+<a href="#lro-status-monitor-no-resource-result" name="lro-status-monitor-no-resource-result">:no_entry:</a> **DO NOT** include a `result` property in the status monitor for a long-running operation that is not a POST-Action long-running operation.
 
 <a href="#lro-status-monitor-retention" name="lro-status-monitor-retention">:white_check_mark:</a> **DO** retain the status monitor resource for some publicly documented period of time (at least 24 hours) after the operation completes.
 
+<a href="#vlro" name="vlro"></a>
+### Very Long-Running Operations (VLROs)
+
+A _very long-running operation (VLRO)_ is an operation that should execute asynchronously. Typically, a VLRO can be identified when 1 or more logical clients need to get the result. For example, if one client initiates the operation and a different browser/client needs the result. An example would be a dashboard or portal that shows all the operations along with their status. In fact, the VLRO pattern offers a way to list all the operations while the LRO pattern does not. Also, the VLRO pattern does not expect a client to make no forward progress due to polling for the long period of time it takes to complete of the operation.
+
+The VLRO pattern is very simple to implement.
+<a href="#vlro-create-update" name="vlro-create-update">:white_check_mark:</a> **DO** implement a PATCH operation to create/update a VLRO service resource. The request/response schemas are identical. The client's request must specify any properties required to create/update the VLRO operation. The response includes the operations's properties as well as the status monitor properties. 
```

Comment:
What is a "VLRO service resource". I think this is the first mention in this file. This should be clearly defined above before we refer to it here.